app/models/project.rb

```diff
@@ -1,3 +1,17 @@
 class Project < ApplicationRecord
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def project_challenge_theme
+    Challenge.find(self.challenge_id).theme
+  end
+
+  def contestant_count
+    Contestant.joins(:projects).where("projects.id = #{self.id}").count
+  end
+
+  def average_contestant_experience
+    Contestant.joins(:projects).where("projects.id = #{self.id}").average("contestants.years_of_experience").round(1)
```

Comment:
`self.contestants.average(:years_of_experience)`

The `self` is optional for these so you can drop and just go straight to contestants if you'd like

Also, there's some grey area around whether `.round` should be in the view or the model. I am personally pretty flexible on it, but in this case it's sort of display logic because you're maybe just rounding so the page looks nice, but you might otherwise want the most precise calculation. There are definitely some devs that strictly believe it's display logic and should live in the view. I think there are business reasons why you might need round in the model (i.e. when handling US currency, I'd want things rounded to the nearest penny no matter what, not just for display purposes). 

Just wanted to share in case you ever get that feedback! There's some grey area with smaller MVC stuff like this which is why we allow for one MVC infraction in the rubric.